README.md

```diff
@@ -282,7 +282,7 @@ If you create a loader or plugin, we would <3 for you to open source it, and put
 
 We consider webpack to be a low-level tool used not only individually but also layered beneath other awesome tools. Because of it's flexibility, webpack isn't always the _easiest_ entry-level solution, however we do believe it is the most powerful. That said, we're always looking for ways improve and simplify the tool without compromising functionality. If you have any ideas on ways to accomplish this, we're all ears!
 
-If you're just getting started, take a look at [our new docs and concepts page](https://webpack.js.org/concepts/). This has a high level overview that is great for beginners!!
+If you're just getting started, take a look at [our new docs and concepts page](https://webpack.js.org/concepts/). This has a high level overview that is great for beginners!! If you are looking for the (old) webpack 1.0 docs please look at these: [Loader-specification](https://github.com/webpack/webpack/wiki/Loader-Specification), [tutorial](https://github.com/webpack/webpack/wiki/Tutorial/1037ac5c91937d130fb0a153b5a7e857b59b093e), [plugins](https://github.com/webpack/webpack/wiki/Plugins/8f90c0b8caa6892ab80add5a40fae5c7b14a87a1), [function](https://github.com/webpack/webpack/wiki/Function).
```

Comment:
Could you do the following edits?

- add a line break after `beginners!!`
- remove `(old)`
- change `functions` link to include the hash